### ISSUE TITLE
Enable HTTPS on community website

### ISSUE THREAD

**Issue by chris001** [OfflineIMAP/imapfw.github.io#4]
Enable HTTPS on the `imapfw` community web site is free.  Someone with access to "Settings :gear:" on this repo, @nicolas33 , could you do this:

1. Under repo name, click "Settings :gear:"
2. In the "Code and automation" section of the sidebar, click "Pages".
3. Scroll down to the custom domain name. 
4. An automatic `DNS` check begins. This check determines if our `DNS` settings are configured to allow GitHub to obtain a certificate automatically. If the check is successful, GitHub queues a job to request a `TLS` certificate from `Let's Encrypt`. On receiving a valid certificate, GitHub automatically uploads it to the GitHub servers that handle `TLS` termination for Pages. When this process completes successfully, a check mark is displayed beside your custom domain name.

The process may take some time. If the process has not completed several minutes after you clicked Save for your custom domain name, try clicking Remove next to your custom domain name. Retype the domain name and click Save again. This will cancel and restart the `TLS` certificate provisioning process.  When `TLS` certificate provisioning is complete, the `Enforce HTTPS` checkbox is accessible.
5. Click to enable the checkbox `Enforce HTTPS`.

**Comment by thekix** [comment]
I think now is done. Please @chris001 , could you test it?

Thanks a lot,
Best Regards,
kix

**Comment by chris001** [comment]
Yes this URL is working now @thekix thanks! https://imapfw.offlineimap.org/
imapfwdoc on gitbook is one of the only remaining broken links!
Any idea what's the gitbook URL for imapfwdoc ?

**Comment by thekix** [comment]
Hi,

the website is https://imapfwdoc.offlineimap.org/?

Probably is https://github.com/OfflineIMAP/imapfw-documentation , but there are not info configured in the Pages section: GitHub Pages is currently disabled.

How can I help?
Best Regards,
kix

**Comment by chris001** [comment]
@nicolas33 is hosting this [imapfw-documentation ](https://github.com/OfflineIMAP/imapfw-documentation) on https://gitbook.com   It's different format from github pages, and a separate company from github.  Someone with access, I guess Nicolas, has to login to gitbook and get the unique public URL of the docs, for the custom https://imapfwdoc.offlineimap.org to alias to.
